src/presentation/test/integration/registration/general-partner-choice.test.ts

```diff
@@ -73,4 +74,19 @@ describe("General Partner Choice Page", () => {
       },
     });
   });
+
+  it("should load the partnership name and name ending with data from api", async () => {
```

Comment:
can you update the tests name by something similar to that:
It should contain the proposed name - data from api

Reply:
test name updated